scripts/produceLKG.mjs

```diff
@@ -15,6 +15,7 @@ const dest = path.join(root, "lib");
 
 async function produceLKG() {
     console.log(`Building LKG from ${source} to ${dest}`);
+    await fs.mkdirp(dest);

```

Comment:
Seems weird that this is also on line 20 but maybe that's intentional; just observing.

Reply:
Probably the latter can go away; it has to at least come first because `del` behaves poorly if the root doesn't exist. I'll test.

Reply:
Actually, this code is pointless now. I should just delete the dir and recreate it. It won't have a README either.

Reply:
Changed. This was probably a side effect of me hacking to make it work many months ago and then never coming back to it until now, now that this is apparently happening and in the milestone 😅

Reply:
All I really want is a type system for describing whether various make dir / delete dir / copy to dir functions fail/don't fail when the relevant target is already there / not there yet / already deleted 🫠